packages-native/bottom-sheet/src/BottomSheet.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Could you go through with Sepideh or Danny about the descriptions ? Because for sure we will get some flags from Connor when we wanna add docs about the props :)